ledger/participant-integration-api/src/main/scala/platform/apiserver/services/ApiSubmissionService.scala

```diff
@@ -258,6 +258,9 @@ private[apiserver] final class ApiSubmissionService private[services] (
             if (submissionDelay.isNegative)
               submitTransaction(transactionInfo)
             else {
+              logger.debug(
```

Comment:
Every submission already produces a log line at info level (see `logger.info("Submitting transaction")` above). I think info level output would be good as well.

Did you also check whether the logging context already contains the command ID? If yes (and I think it does), then you could simply write `"Delaying submission by $submissionDelay"`.

Reply:
You are right commandId is already include, so trimming down the text portion and increasing the log level to info. Thanks!